rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/executor/BakePoller.groovy

```diff
@@ -131,8 +131,9 @@ class BakePoller implements ApplicationListener<ContextRefreshedEvent> {
 
                             // This will have the most up-to-date timestamp.
                             bakeStatus = bakeStore.retrieveBakeStatusById(statusId)
-                            Id failedBakesId = registry.createId('bakes').withTag("success", "false").withTag("cause", "orphanTimedOut")
-                            registry.timer(failedBakesId).record(bakeStatus.updatedTimestamp - bakeStatus.createdTimestamp, TimeUnit.MILLISECONDS)
+                            def tags = ["success":"false", "cause":"orphanTimedOut", "region":bakeStore.retrieveRegionById(statusId)]
```

Comment:
Also don't need quotes around the map keys:

`[success: "false", cause: "orphanTimedOut", ...]`